conf/viewvc.conf.dist

```diff
@@ -114,8 +114,9 @@
 ## root_parents: Specifies a list of directories under which any
 ## number of repositories may reside.  You can specify multiple root
 ## parents separated by commas or new lines, each of which is of the
-## form "path: type" (where the type is either "cvs" or "svn", and
-## the path is an absolute filesystem path).
+## form "path [= context] : type" (where the type is either "cvs" or
+## "svn", the path is an absolute filesystem path, and the (optional)
+## context is prepended to each repository's name).
 ##
```

Comment:
Good point. For the 'terminology' section, would something like this make it clear enough?
```
##    "context" - Contexts are a concept for grouping together several
##        repositories, similar to using (sub)directories.  For a root name
##        containing a slash (/), the context is defined as the part of the
##        root name before the (last) slash.  A root name without a slash is
##        said to have an empty or no context.
```